scripts/scil_bundle_fixel_analysis.py

```diff
@@ -0,0 +1,363 @@
+                        'its filename without extensions.')
+
+    p.add_argument('--dps_key', default=None, type=str,
+                   help='Key to access the data per streamline to use as '
+                        'weight when computing the maps, \ninstead of the '
+                        'number of streamlines. [%(default)s].')
+
+    p.add_argument('--max_theta', default=45,
+                   help='Maximum angle between streamline and peak to be '
+                        'associated [%(default)s].')
+
+    g1 = p.add_argument_group(title='Mask parameters')
+
+    g1.add_argument('--abs_thr', default=0, type=float,
+                    help='Value of density maps threshold to obtain density '
+                         'masks, in number of streamlines \nor streamline '
+                         'weighting if --dps_key is given. Any number of '
+                         'streamlines \nor weight above this value will '
+                         'pass the absolute threshold test [%(default)s].')
+
+    g1.add_argument('--rel_thr', default=0.01, type=float,
+                    help='Value of density maps threshold to obtain density '
+                         'masks, as a ratio of the normalized density '
+                         '\nAny normalized density above '
+                         'this value will pass the relative threshold test. '
+                         '\nMust be between 0 and 1 [%(default)s].')
+
+    g1.add_argument('--norm', default="voxel", choices=["fixel", "voxel"],
+                    help='Way of normalizing the density maps. If fixel, '
+                         'will normalize the maps per fixel, \nin each voxel. '
+                         'If voxel, will normalize the maps per voxel '
+                         '[%(default)s].')
+
+    g2 = p.add_argument_group(title='Output options')
+
+    g2.add_argument('--split_bundles', action='store_true',
+                    help='If set, save the density maps for each bundle '
+                         'separately \nin addition to the all in one version.')
+
+    g2.add_argument('--split_fixels', action='store_true',
+                    help='If set, save the density maps for each fixel '
+                         'separately \nin addition to the all in one version.')
+
+    g2.add_argument('--single_bundle', action='store_true',
+                    help='If set, will save the single-fiber single-bundle '
+                         'masks as well. \nThese are obtained by '
+                         'selecting the voxels where only one bundle is '
+                         'present \n(and one fiber/fixel).')
+
+    g2.add_argument('--bundles_mask', action='store_true',
+                    help='If set, save the bundle mask for each bundle.')
+
+    g2.add_argument('--out_dir', default="./",
+                    help='Path to the output directory where all the output '
+                         'files will be saved [%(default)s].')
```

Comment:
Please run the mkdir if the --out_dir does not exist.

Reply:
It just crashed on me because the out_dir did not exist after 10 min of computation. ... grrrr...

Reply:
@arnaudbore what is the typical scilpy way of dealing with an output directory?